script/1_0_2current_en_script/Arcueid/Day 5/05_00_ARC05_3_2B.txt

```diff
@@ -27,58 +27,58 @@ I'll be the decoy, and you'll be the attacker. If it's you, then the question of
 Leaving me aside, you're the one that has to fight that guy head-on. What are you going to do about those hand-like flames?"
 <Page9>
 "Hm? That shouldn't be a problem.
-After taking that hit, I figured that Vlov's flame was somewhere around 3000 <C|o   >. So I'll be able to endure if I rebuild my body to become heat resistant from now on."
+After taking that hit, I figured that Vlov's flame was somewhere around 3000< |o>C. So I'll be able to endure if I rebuild my body to become heat resistant from now on."
 <Page10>
 "Huh...?"
-No, isn't it weird to say a number like 3000 <C|o   > so casually?
+No, isn't it weird to say a number like 3000< |o>C so casually?
 Wouldn't that kind of temperature be enough to melt even a rock?
 <Page11>
 "The surrounding flames are much easier to handle.
-Those only cover the surface, so when you come out of the shadows, I'll give off a little scream and blow them away.
+Those only cover the surface, so when you come out of the shadows, I'll give off a little scream and blow them away.//to be rechecked
```

Comment:
I imagined this line from the Japanese as her like announcing her move like you see in old mecha anime but I wasn't sure how to phrase that and I think left Tumbling's version. For now I think "shout" is an improvement but otherwise leave it.